### PR TITLE
Fixes #12109: Agent capabilities in inventory are not parsed

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/12109

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/components/schemas/node-full.yml [hunk 1]
```diff
@@ -314,6 +314,20 @@ properties:
...
+        items:
+          type: string
+          description: Special capacities of the agent, like parsing xml or yaml, support of advanced acl, etc.
+          example: xml
+      serverRoles:
+        type: array
+        description: Local user account used by the agent
+        items:
+          type: string
+          description: A role that node fullfill in management technology
```
Comment (amousset) [inline webapp/sources/api-doc/components/schemas/node-full.yml:329]: fullfills

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**